test/README.md

```diff
@@ -0,0 +1,48 @@
+# Torchaudio Test Suite
+
+## Structure of tests
+
+The followings are the categories of tests (and corresponding test modules) for `torchaudio`.
```

Comment:
nit: following

The following is an overview of the tests and related modules for `torchaudio`.